lib/wallet/wallet.js

```diff
@@ -82,9 +82,15 @@ class Wallet extends EventEmitter {
 
     const bidsCacheSizeInBlocks = 20;
     const bidsPerAuction = 3; // average in March 2020 is ~3.1
-    const capacity =
+
+    // TODO (bennett) - make this configurable from options
+    const bidsCapacity =
       consensus.MAX_BLOCK_OPENS * bidsPerAuction * bidsCacheSizeInBlocks;
-    this.sendBidResults = new LRU(capacity);
+    this.sendBidResults = new LRU(bidsCapacity);
+
+    // TODO (bennett) - make this configurable from options
+    const opensCapacity = 6500; // 300 opens * 20 + 1 blocks + 200 extra
```

Comment:
Would prefer if the math were just coded in so we can change the parameters without needing to do the math ourselves (eg when we change 20 to 6)